apps/notifications/src/panel/state/selectors/get-keyboard-shortcuts-enabled.js

```diff
@@ -0,0 +1 @@
+export default ( state ) => state.ui?.keyboardShortcutsAreEnabled;
```

Comment:
hm. this is a `.js` file but it's using optional chaining. should we rename it to `.ts`?

Reply:
Hm, is that mutually exclusive? It is mentioned in the [JavaScript Coding Guidelines](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/javascript.md) but without a relation to TypeScript and I've seen many `.js` files using optional chaining throughout Calypso.

Reply:
Calypso build's babel config supports optional chaining in `.jsx?` files since #37552, so that's totally fine to use like this.

Reply:
I guess that @dmsnell hasn't used plain JavaScript in the last few months, since the standard got some new features 😉 SimpleNote is 100% TypeScript and Erlang?

Reply:
Well I didn't realize it was a stage 4 proposal to ECMAScript so that's cool

> SimpleNote is 100% TypeScript and Erlang?

The Electron-specific code is still not transpiled so it relies on whatever Chrome/V8 can natively run. The client application though is all run through TypeScript. We made that jump with Simplenote and it worked out great for us. Simplenote doesn't contain any Erlang, though it's possible that at some point Erlang code will replace some Python code that powers parts of Simplenote, including published notes and sharing notes with other users.